.python-version

```diff
@@ -1 +1,2 @@
-2.7.14
+2.7.6
+3.8.3
```

Comment:
3.8.4 ?!?  I know how difficult it is to upgrade things once they are set.

Reply:
OH DARN! IT WAS RELEASED YESTERDAY!! :D I actually check to make sure I got the latest version when I wrote _code--_also_ yesterday :P Python3 will only ever exist inside docker, so updating should be easy from now on :)

Reply:
Let's do it in a future PR I think; it takes too long to do these sort of changes and rebuild and test that it's working.